std_msgs/Int64 name
string state

